backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/navigationbar/internal/BpkTopNavBarImpl.kt

```diff
@@ -66,17 +66,23 @@ internal fun BpkTopNavBarImpl(
     val fraction = 1f - internalState.state.collapsedFraction
 
     val backgroundColor = when {
-        fraction <= 0f -> BpkTheme.colors.surfaceDefault
+        fraction <= 0f -> when(style) {
+            NavBarStyle.OnDark -> BpkTheme.colors.surfaceContrast
+            else -> BpkTheme.colors.surfaceDefault
+        }
         else -> when (style) {
-            NavBarStyle.OnImage -> Color.Transparent
+            NavBarStyle.OnImage, NavBarStyle.OnDark -> Color.Transparent
```

Comment:
shouldn't it also be `surfaceContrast` here? it is just a solid case

Reply:
Yeah, let's make it a solid case. I wasn't 100% sure when checking the designs, but done now